bitrise.yml

```diff
@@ -0,0 +1,172 @@
+---
+format_version: '11'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+workflows:
+  DeployFirebase:
+    description: |
+      The workflow builds and deploys the app in Firebase and Bitrise step.
+
+      For testing the *retry_on_failure* test repetition mode is enabled.
+
+      Next steps:
+      - Set up [Connecting to an Apple service with API key](https://devcenter.bitrise.io/en/accounts/connecting-to-services/connecting-to-an-apple-service-with-api-key.html##).
+      - Or further customise code signing following our [iOS code signing](https://devcenter.bitrise.io/en/code-signing/ios-code-signing.html) guide.
+    steps:
+    - authenticate-host-with-netrc@0:
+        inputs:
+        - username: "$GITHUB_USER_NAME"
+        - password: "$GITHUB_PASSWORD"
+        - host: github.com
+    - git-clone@6: {}
+    - cache-pull@2: {}
+    - cocoapods-install@2: {}
+    - fastlane-match@0:
```

Comment:
I also feel better if we have custom fastlane's lane. This is going to be duplicated a lot if we have more workflows

Reply:
Main purpose of this objective is to use `Bitrise` so I use `Bitrise built in steps` as much as possible.

Reply:
I agree with SuV, if we use fastlane you'll have more freedom of moving to another CICD or use a local machine as CICD. You can think about it as another feature in future PRs, if you are interested.

Reply:
You are right. Currently I am working on `Fastlane` in my IC project and may be after first release of this app I can include that here too.